2017-12-29 07:47:24,077 INFO  o.h.validator.internal.util.Version - HV000001: Hibernate Validator 5.2.2.Final
2017-12-29 07:47:24,202 INFO  o.s.c.a.AnnotationConfigApplicationContext - Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@7919964f: startup date [Fri Dec 29 07:47:24 UTC 2017]; root of context hierarchy
2017-12-29 07:47:24,443 INFO  o.s.b.f.a.AutowiredAnnotationBeanPostProcessor - JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
2017-12-29 07:47:24,569 INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'configurationPropertiesRebinderAutoConfiguration' of type [class org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$5657a801] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2017-12-29 07:47:25,041 INFO  o.s.c.c.c.ConfigServicePropertySourceLocator - Fetching config from server at: http://localhost:8888
2017-12-29 07:47:25,239 WARN  o.s.c.c.c.ConfigServicePropertySourceLocator - Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused (Connection refused); nested exception is java.net.ConnectException: Connection refused (Connection refused)
2017-12-29 07:47:25,241 INFO  com.capitalone.dashboard.Application - No profiles are active
2017-12-29 07:47:25,274 INFO  o.s.c.a.AnnotationConfigApplicationContext - Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@1f63e00d: startup date [Fri Dec 29 07:47:25 UTC 2017]; parent: org.springframework.context.annotation.AnnotationConfigApplicationContext@7919964f
2017-12-29 07:47:26,808 INFO  o.s.cloud.context.scope.GenericScope - BeanFactory id=5ae94b21-555c-3e34-89ac-4b23bd02e833
2017-12-29 07:47:26,813 INFO  c.u.j.EnableEncryptablePropertySourcesPostProcessor - Post-processing PropertySource instances
2017-12-29 07:47:26,824 INFO  c.u.j.c.StringEncryptorConfiguration - String Encryptor custom Bean not found with name 'jasyptStringEncryptor'. Initializing String Encryptor based on properties with name 'jasyptStringEncryptor'
2017-12-29 07:47:26,830 INFO  c.u.j.EnableEncryptablePropertySourcesPostProcessor - Converting PropertySource commandLineArgs [org.springframework.core.env.SimpleCommandLinePropertySource] to EncryptableEnumerablePropertySourceWrapper
2017-12-29 07:47:26,831 INFO  c.u.j.EnableEncryptablePropertySourcesPostProcessor - Converting PropertySource systemProperties [org.springframework.core.env.MapPropertySource] to EncryptableMapPropertySourceWrapper
2017-12-29 07:47:26,831 INFO  c.u.j.EnableEncryptablePropertySourcesPostProcessor - Converting PropertySource systemEnvironment [org.springframework.core.env.SystemEnvironmentPropertySource] to EncryptableMapPropertySourceWrapper
2017-12-29 07:47:26,831 INFO  c.u.j.EnableEncryptablePropertySourcesPostProcessor - Converting PropertySource random [org.springframework.boot.context.config.RandomValuePropertySource] to EncryptablePropertySourceWrapper
2017-12-29 07:47:26,831 INFO  c.u.j.EnableEncryptablePropertySourcesPostProcessor - Converting PropertySource applicationConfig: [file:application.properties] [org.springframework.core.env.PropertiesPropertySource] to EncryptableMapPropertySourceWrapper
2017-12-29 07:47:26,831 INFO  c.u.j.EnableEncryptablePropertySourcesPostProcessor - Converting PropertySource springCloudClientHostInfo [org.springframework.core.env.MapPropertySource] to EncryptableMapPropertySourceWrapper
2017-12-29 07:47:26,832 INFO  c.u.j.EnableEncryptablePropertySourcesPostProcessor - Converting PropertySource defaultProperties [org.springframework.core.env.MapPropertySource] to EncryptableMapPropertySourceWrapper
2017-12-29 07:47:26,835 INFO  o.s.b.f.a.AutowiredAnnotationBeanPostProcessor - JSR-330 'javax.inject.Inject' annotation found and supported for autowiring
2017-12-29 07:47:26,873 INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker - Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [class org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$5657a801] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2017-12-29 07:47:27,405 INFO  o.s.s.c.ThreadPoolTaskScheduler - Initializing ExecutorService  'taskScheduler'
2017-12-29 07:47:27,491 INFO  c.c.dashboard.config.MongoConfig - ReplicaSetfalse
2017-12-29 07:47:27,598 INFO  c.c.dashboard.config.MongoConfig - Initializing Mongo Client server at: localhost:27017
2017-12-29 07:47:27,637 INFO  org.mongodb.driver.cluster - Cluster created with settings {hosts=[localhost:27017], mode=SINGLE, requiredClusterType=UNKNOWN, serverSelectionTimeout='30000 ms', maxWaitQueueSize=500}
2017-12-29 07:47:27,735 INFO  c.c.dashboard.config.MongoConfig - Connecting to Mongo: Mongo{options=MongoClientOptions{description='null', readPreference=primary, writeConcern=WriteConcern{w=1, wtimeout=0, fsync=false, j=false, codecRegistry=org.bson.codecs.configuration.ProvidersCodecRegistry@c994fcc0, minConnectionsPerHost=0, maxConnectionsPerHost=100, threadsAllowedToBlockForConnectionMultiplier=5, serverSelectionTimeout=30000, maxWaitTime=120000, maxConnectionIdleTime=60000, maxConnectionLifeTime=0, connectTimeout=10000, socketTimeout=0, socketKeepAlive=false, sslEnabled=false, sslInvalidHostNamesAllowed=false, alwaysUseMBeans=false, heartbeatFrequency=10000, minHeartbeatFrequency=500, heartbeatConnectTimeout=20000, heartbeatSocketTimeout=20000, localThreshold=15, requiredReplicaSetName='null', dbDecoderFactory=com.mongodb.DefaultDBDecoder$1@38d3db47, dbEncoderFactory=com.mongodb.DefaultDBEncoder$1@3f7c9c3e, socketFactory=javax.net.DefaultSocketFactory@329cae36, cursorFinalizerEnabled=true, connectionPoolSettings=ConnectionPoolSettings{maxSize=100, minSize=0, maxWaitQueueSize=500, maxWaitTimeMS=120000, maxConnectionLifeTimeMS=0, maxConnectionIdleTimeMS=60000, maintenanceInitialDelayMS=0, maintenanceFrequencyMS=60000}, socketSettings=SocketSettings{connectTimeoutMS=10000, readTimeoutMS=0, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}, serverSettings=ServerSettings{heartbeatFrequencyMS=10000, minHeartbeatFrequencyMS=500}, heartbeatSocketSettings=SocketSettings{connectTimeoutMS=20000, readTimeoutMS=20000, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}}}
2017-12-29 07:47:28,145 INFO  org.mongodb.driver.cluster - No server chosen by PrimaryServerSelector from cluster description ClusterDescription{type=UNKNOWN, connectionMode=SINGLE, all=[ServerDescription{address=localhost:27017, type=UNKNOWN, state=CONNECTING}]}. Waiting for 30000 ms before timing out
2017-12-29 07:47:28,162 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:1, serverValue:319}] to localhost:27017
2017-12-29 07:47:28,163 INFO  org.mongodb.driver.cluster - Monitor thread successfully connected to server with description ServerDescription{address=localhost:27017, type=STANDALONE, state=CONNECTED, ok=true, version=ServerVersion{versionList=[3, 6, 1]}, minWireVersion=0, maxWireVersion=6, electionId=null, maxDocumentSize=16777216, roundTripTimeNanos=512804}
2017-12-29 07:47:28,217 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:2, serverValue:320}] to localhost:27017
2017-12-29 07:47:28,620 INFO  c.c.dashboard.config.MongoConfig - ReplicaSetfalse
2017-12-29 07:47:28,620 INFO  c.c.dashboard.config.MongoConfig - Initializing Mongo Client server at: localhost:27017
2017-12-29 07:47:28,621 INFO  org.mongodb.driver.cluster - Cluster created with settings {hosts=[localhost:27017], mode=SINGLE, requiredClusterType=UNKNOWN, serverSelectionTimeout='30000 ms', maxWaitQueueSize=500}
2017-12-29 07:47:28,623 INFO  c.c.dashboard.config.MongoConfig - Connecting to Mongo: Mongo{options=MongoClientOptions{description='null', readPreference=primary, writeConcern=WriteConcern{w=1, wtimeout=0, fsync=false, j=false, codecRegistry=org.bson.codecs.configuration.ProvidersCodecRegistry@c994fcc0, minConnectionsPerHost=0, maxConnectionsPerHost=100, threadsAllowedToBlockForConnectionMultiplier=5, serverSelectionTimeout=30000, maxWaitTime=120000, maxConnectionIdleTime=60000, maxConnectionLifeTime=0, connectTimeout=10000, socketTimeout=0, socketKeepAlive=false, sslEnabled=false, sslInvalidHostNamesAllowed=false, alwaysUseMBeans=false, heartbeatFrequency=10000, minHeartbeatFrequency=500, heartbeatConnectTimeout=20000, heartbeatSocketTimeout=20000, localThreshold=15, requiredReplicaSetName='null', dbDecoderFactory=com.mongodb.DefaultDBDecoder$1@38d3db47, dbEncoderFactory=com.mongodb.DefaultDBEncoder$1@3f7c9c3e, socketFactory=javax.net.DefaultSocketFactory@329cae36, cursorFinalizerEnabled=true, connectionPoolSettings=ConnectionPoolSettings{maxSize=100, minSize=0, maxWaitQueueSize=500, maxWaitTimeMS=120000, maxConnectionLifeTimeMS=0, maxConnectionIdleTimeMS=60000, maintenanceInitialDelayMS=0, maintenanceFrequencyMS=60000}, socketSettings=SocketSettings{connectTimeoutMS=10000, readTimeoutMS=0, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}, serverSettings=ServerSettings{heartbeatFrequencyMS=10000, minHeartbeatFrequencyMS=500}, heartbeatSocketSettings=SocketSettings{connectTimeoutMS=20000, readTimeoutMS=20000, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}}}
2017-12-29 07:47:28,644 INFO  c.c.dashboard.config.MongoConfig - ReplicaSetfalse
2017-12-29 07:47:28,644 INFO  c.c.dashboard.config.MongoConfig - Initializing Mongo Client server at: localhost:27017
2017-12-29 07:47:28,645 INFO  org.mongodb.driver.cluster - Cluster created with settings {hosts=[localhost:27017], mode=SINGLE, requiredClusterType=UNKNOWN, serverSelectionTimeout='30000 ms', maxWaitQueueSize=500}
2017-12-29 07:47:28,658 INFO  c.c.dashboard.config.MongoConfig - Connecting to Mongo: Mongo{options=MongoClientOptions{description='null', readPreference=primary, writeConcern=WriteConcern{w=1, wtimeout=0, fsync=false, j=false, codecRegistry=org.bson.codecs.configuration.ProvidersCodecRegistry@c994fcc0, minConnectionsPerHost=0, maxConnectionsPerHost=100, threadsAllowedToBlockForConnectionMultiplier=5, serverSelectionTimeout=30000, maxWaitTime=120000, maxConnectionIdleTime=60000, maxConnectionLifeTime=0, connectTimeout=10000, socketTimeout=0, socketKeepAlive=false, sslEnabled=false, sslInvalidHostNamesAllowed=false, alwaysUseMBeans=false, heartbeatFrequency=10000, minHeartbeatFrequency=500, heartbeatConnectTimeout=20000, heartbeatSocketTimeout=20000, localThreshold=15, requiredReplicaSetName='null', dbDecoderFactory=com.mongodb.DefaultDBDecoder$1@38d3db47, dbEncoderFactory=com.mongodb.DefaultDBEncoder$1@3f7c9c3e, socketFactory=javax.net.DefaultSocketFactory@329cae36, cursorFinalizerEnabled=true, connectionPoolSettings=ConnectionPoolSettings{maxSize=100, minSize=0, maxWaitQueueSize=500, maxWaitTimeMS=120000, maxConnectionLifeTimeMS=0, maxConnectionIdleTimeMS=60000, maintenanceInitialDelayMS=0, maintenanceFrequencyMS=60000}, socketSettings=SocketSettings{connectTimeoutMS=10000, readTimeoutMS=0, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}, serverSettings=ServerSettings{heartbeatFrequencyMS=10000, minHeartbeatFrequencyMS=500}, heartbeatSocketSettings=SocketSettings{connectTimeoutMS=20000, readTimeoutMS=20000, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}}}
2017-12-29 07:47:28,679 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:3, serverValue:321}] to localhost:27017
2017-12-29 07:47:28,680 INFO  org.mongodb.driver.cluster - Monitor thread successfully connected to server with description ServerDescription{address=localhost:27017, type=STANDALONE, state=CONNECTED, ok=true, version=ServerVersion{versionList=[3, 6, 1]}, minWireVersion=0, maxWireVersion=6, electionId=null, maxDocumentSize=16777216, roundTripTimeNanos=322603}
2017-12-29 07:47:28,700 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:4, serverValue:322}] to localhost:27017
2017-12-29 07:47:28,701 INFO  org.mongodb.driver.cluster - Monitor thread successfully connected to server with description ServerDescription{address=localhost:27017, type=STANDALONE, state=CONNECTED, ok=true, version=ServerVersion{versionList=[3, 6, 1]}, minWireVersion=0, maxWireVersion=6, electionId=null, maxDocumentSize=16777216, roundTripTimeNanos=320903}
2017-12-29 07:47:29,145 INFO  c.c.dashboard.config.MongoConfig - ReplicaSetfalse
2017-12-29 07:47:29,146 INFO  c.c.dashboard.config.MongoConfig - Initializing Mongo Client server at: localhost:27017
2017-12-29 07:47:29,148 INFO  org.mongodb.driver.cluster - Cluster created with settings {hosts=[localhost:27017], mode=SINGLE, requiredClusterType=UNKNOWN, serverSelectionTimeout='30000 ms', maxWaitQueueSize=500}
2017-12-29 07:47:29,160 INFO  c.c.dashboard.config.MongoConfig - Connecting to Mongo: Mongo{options=MongoClientOptions{description='null', readPreference=primary, writeConcern=WriteConcern{w=1, wtimeout=0, fsync=false, j=false, codecRegistry=org.bson.codecs.configuration.ProvidersCodecRegistry@c994fcc0, minConnectionsPerHost=0, maxConnectionsPerHost=100, threadsAllowedToBlockForConnectionMultiplier=5, serverSelectionTimeout=30000, maxWaitTime=120000, maxConnectionIdleTime=60000, maxConnectionLifeTime=0, connectTimeout=10000, socketTimeout=0, socketKeepAlive=false, sslEnabled=false, sslInvalidHostNamesAllowed=false, alwaysUseMBeans=false, heartbeatFrequency=10000, minHeartbeatFrequency=500, heartbeatConnectTimeout=20000, heartbeatSocketTimeout=20000, localThreshold=15, requiredReplicaSetName='null', dbDecoderFactory=com.mongodb.DefaultDBDecoder$1@38d3db47, dbEncoderFactory=com.mongodb.DefaultDBEncoder$1@3f7c9c3e, socketFactory=javax.net.DefaultSocketFactory@329cae36, cursorFinalizerEnabled=true, connectionPoolSettings=ConnectionPoolSettings{maxSize=100, minSize=0, maxWaitQueueSize=500, maxWaitTimeMS=120000, maxConnectionLifeTimeMS=0, maxConnectionIdleTimeMS=60000, maintenanceInitialDelayMS=0, maintenanceFrequencyMS=60000}, socketSettings=SocketSettings{connectTimeoutMS=10000, readTimeoutMS=0, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}, serverSettings=ServerSettings{heartbeatFrequencyMS=10000, minHeartbeatFrequencyMS=500}, heartbeatSocketSettings=SocketSettings{connectTimeoutMS=20000, readTimeoutMS=20000, keepAlive=false, receiveBufferSize=0, sendBufferSize=0}}}
2017-12-29 07:47:29,198 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:5, serverValue:323}] to localhost:27017
2017-12-29 07:47:29,207 INFO  org.mongodb.driver.cluster - Monitor thread successfully connected to server with description ServerDescription{address=localhost:27017, type=STANDALONE, state=CONNECTED, ok=true, version=ServerVersion{versionList=[3, 6, 1]}, minWireVersion=0, maxWireVersion=6, electionId=null, maxDocumentSize=16777216, roundTripTimeNanos=8699070}
2017-12-29 07:47:30,074 INFO  o.s.j.e.a.AnnotationMBeanExporter - Registering beans for JMX exposure on startup
2017-12-29 07:47:30,085 INFO  o.s.j.e.a.AnnotationMBeanExporter - Bean with name 'configurationPropertiesRebinder' has been autodetected for JMX exposure
2017-12-29 07:47:30,087 INFO  o.s.j.e.a.AnnotationMBeanExporter - Bean with name 'environmentManager' has been autodetected for JMX exposure
2017-12-29 07:47:30,087 INFO  o.s.j.e.a.AnnotationMBeanExporter - Bean with name 'refreshScope' has been autodetected for JMX exposure
2017-12-29 07:47:30,091 INFO  o.s.j.e.a.AnnotationMBeanExporter - Located managed bean 'environmentManager': registering with JMX server as MBean [org.springframework.cloud.context.environment:name=environmentManager,type=EnvironmentManager]
2017-12-29 07:47:30,108 INFO  o.s.j.e.a.AnnotationMBeanExporter - Located managed bean 'refreshScope': registering with JMX server as MBean [org.springframework.cloud.context.scope.refresh:name=refreshScope,type=RefreshScope]
2017-12-29 07:47:30,118 INFO  o.s.j.e.a.AnnotationMBeanExporter - Located managed bean 'configurationPropertiesRebinder': registering with JMX server as MBean [org.springframework.cloud.context.properties:name=configurationPropertiesRebinder,context=1f63e00d,type=ConfigurationPropertiesRebinder]
2017-12-29 07:47:30,223 INFO  com.capitalone.dashboard.Application - Started Application in 6.708 seconds (JVM running for 7.569)
2017-12-29 07:49:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:2, serverValue:320}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 07:50:00,001 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 07:50:00,306 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:6, serverValue:328}] to localhost:27017
2017-12-29 07:50:00,365 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 07:50:00,365 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 07:50:00,365 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 07:50:00,440 INFO  c.c.d.collector.CollectorTask - Repo Count              0        0s
2017-12-29 07:50:00,440 INFO  c.c.d.collector.CollectorTask - New Commits             0        0s
2017-12-29 07:50:00,440 INFO  c.c.d.collector.CollectorTask - Finished                         0s
2017-12-29 07:51:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:6, serverValue:328}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 07:55:00,001 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 07:55:00,069 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:7, serverValue:337}] to localhost:27017
2017-12-29 07:55:00,082 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 07:55:00,083 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 07:55:00,083 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 07:55:04,057 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 07:55:04,063 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 07:56:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:7, serverValue:337}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:00:00,000 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:00:00,074 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:8, serverValue:347}] to localhost:27017
2017-12-29 08:00:00,130 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:00:00,130 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:00:00,130 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:00:02,564 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:00:02,564 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:01:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:8, serverValue:347}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:05:00,001 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:05:00,047 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:9, serverValue:354}] to localhost:27017
2017-12-29 08:05:00,080 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:05:00,080 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:05:00,080 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:05:02,952 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:05:02,953 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:06:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:9, serverValue:354}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:10:00,000 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:10:00,033 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:10, serverValue:358}] to localhost:27017
2017-12-29 08:10:00,070 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:10:00,070 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:10:00,070 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:10:02,722 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:10:02,726 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:11:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:10, serverValue:358}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:15:00,000 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:15:00,057 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:11, serverValue:370}] to localhost:27017
2017-12-29 08:15:00,071 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:15:00,071 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:15:00,071 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:15:02,909 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:15:02,910 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:16:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:11, serverValue:370}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:20:00,000 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:20:00,037 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:12, serverValue:375}] to localhost:27017
2017-12-29 08:20:00,041 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:20:00,041 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:20:00,041 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:20:02,609 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:20:02,610 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:21:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:12, serverValue:375}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:25:00,000 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:25:00,064 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:13, serverValue:378}] to localhost:27017
2017-12-29 08:25:00,068 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:25:00,068 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:25:00,068 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:25:03,253 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:25:03,253 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:26:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:13, serverValue:378}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:30:00,002 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:30:00,075 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:14, serverValue:383}] to localhost:27017
2017-12-29 08:30:00,086 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:30:00,086 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:30:00,092 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:30:02,675 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:30:02,676 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:31:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:14, serverValue:383}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:35:00,002 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:35:00,033 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:15, serverValue:387}] to localhost:27017
2017-12-29 08:35:00,042 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:35:00,042 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:35:00,042 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:35:03,043 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:35:03,043 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:36:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:15, serverValue:387}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:40:00,001 INFO  c.c.d.collector.CollectorTask - Running Collector: Bitbucket
2017-12-29 08:40:00,071 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:16, serverValue:389}] to localhost:27017
2017-12-29 08:40:00,075 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:40:00,075 INFO  c.c.d.collector.CollectorTask - Starting...
2017-12-29 08:40:00,075 INFO  c.c.d.collector.CollectorTask - -----------------------------------
2017-12-29 08:40:03,269 ERROR c.c.d.c.DefaultBitbucketServerClient - null
2017-12-29 08:40:03,269 ERROR o.s.s.s.TaskUtils$LoggingErrorHandler - Unexpected error occurred in scheduled task.
java.lang.NullPointerException: null
	at com.capitalone.dashboard.collector.DefaultBitbucketServerClient.getCommits(DefaultBitbucketServerClient.java:94) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.GitCollectorTask.collect(GitCollectorTask.java:148) ~[bitbucket-scm-collector-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at com.capitalone.dashboard.collector.CollectorTask.run(CollectorTask.java:63) ~[core-2.0.5-SNAPSHOT.jar!/:2.0.5-SNAPSHOT]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:81) [spring-context-4.2.5.RELEASE.jar!/:4.2.5.RELEASE]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_151]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_151]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_151]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_151]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_151]
2017-12-29 08:41:27,652 INFO  org.mongodb.driver.connection - Closed connection [connectionId{localValue:16, serverValue:389}] to localhost:27017 because it is past its maximum allowed idle time.
2017-12-29 08:43:27,750 INFO  o.s.c.a.AnnotationConfigApplicationContext - Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@1f63e00d: startup date [Fri Dec 29 07:47:25 UTC 2017]; parent: org.springframework.context.annotation.AnnotationConfigApplicationContext@7919964f
2017-12-29 08:43:27,756 INFO  o.s.j.e.a.AnnotationMBeanExporter - Unregistering JMX-exposed beans on shutdown
2017-12-29 08:43:27,757 INFO  o.s.j.e.a.AnnotationMBeanExporter - Unregistering JMX-exposed beans
2017-12-29 08:43:27,784 INFO  org.mongodb.driver.connection - Opened connection [connectionId{localValue:17, serverValue:391}] to localhost:27017
2017-12-29 08:43:27,787 INFO  o.s.s.c.ThreadPoolTaskScheduler - Shutting down ExecutorService 'taskScheduler'
